src/askjsx/lib/Call.tsx

```diff
@@ -15,6 +15,21 @@ export function Call({
 }) {
   assert(isString(name), 'name');
   // assert(isStringArray(args), 'args');
+  if (name === 'useFor') {
```

Comment:
Please add a comment explaining that this is a temporary way of linking some AskScript code. We'll have to make typing work before introducing a better way